atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/postgres/DbKvsPostgresTargetedSweepIntegrationTest.java

```diff
@@ -16,33 +16,18 @@
 
 package com.palantir.atlasdb.keyvalue.dbkvs.impl.postgres;
 
-import com.palantir.atlasdb.keyvalue.impl.TestResourceManager;
-import com.palantir.atlasdb.sweep.AbstractTargetedSweepTest;
+import com.palantir.atlasdb.keyvalue.impl.TestResourceManagerV2;
 import com.palantir.atlasdb.transaction.impl.SweepStrategyManagers.CacheWarming;
-import java.util.Arrays;
-import org.junit.ClassRule;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
-@RunWith(Parameterized.class)
-public class DbKvsPostgresTargetedSweepIntegrationTest extends AbstractTargetedSweepTest {
-    @ClassRule
-    public static final TestResourceManager TRM = new TestResourceManager(DbKvsPostgresTestSuite::createKvs);
+@ExtendWith(DbKvsPostgresExtension.class)
+public class DbKvsPostgresTargetedSweepIntegrationTest extends AbstractDbKvsPostgresTargetedSweepIntegrationTest {
```

Comment:
Can we name this in a more similar way to `CacheWarmingDbKvsPostgresTargetedSweepIntegrationTest`? Something like `NoCacheWarming...` or `CacheWarmingDisabled...`